### ISSUE TITLE
go get fails: fatal error: 'openssl/sha.h' file not found

### ISSUE THREAD

**Issue by montanaflynn** [ctz/go-fastpbkdf2#1]
```
$ go get github.com/ctz/go-fastpbkdf2
# github.com/ctz/go-fastpbkdf2
../Development/go/src/github.com/ctz/go-fastpbkdf2/fastpbkdf2.c:20:10: fatal error: 'openssl/sha.h' file not found

**Comment by kazegawa** [comment]
what ur platform?
when ur using linux. use libssl

**Comment by 199862** [comment]
Hi, same problem, windows 10
C:\Go\src>go get "github.com/ctz/go-fastpbkdf2"
 github.com/ctz/go-fastpbkdf2
fastpbkdf2.c:20:10: fatal error: openssl/sha.h: No such file or directory
include <openssl/sha.h>

**Comment by ctz** [comment]
Yes, this needs openssl to be installed and available on the system you're building on. That'll make using it on Windows quite challenging.

**Comment by bitwrite** [comment]
If you want to run on windows 10 i suggest you change the program to use 
import "golang.org/x/crypto/pbkdf2"
instead of "github.com/ctz/go-fastpbkdf2"

Mines running fine now after compiling with the "golang.org/x/crypto/pbkdf2"

**Comment by bitwrite** [comment]
With regards to the error

fastpbkdf2.c:20:10: fatal error: 'openssl/sha.h' file not found 

I had to install openssl and was still getting the above error.
then install libssl-dev if your still having issues

sudo apt-get install libssl-dev 

This worked for me running on my Raspberry
